pkg/agent/cniserver/pod_configuration.go

```diff
@@ -677,12 +677,21 @@ func (pc *podConfigurator) recordPodEvent(ifConfig *interfacestore.InterfaceConf
 }
 
 func (pc *podConfigurator) processPortStatusMessage(status *openflow15.PortStatus) {
-	// Update Pod OpenFlow entries only after the OpenFlow port state is live.
-	if status.Desc.State != openflow15.PS_LIVE {
+	ofPort := status.Desc.PortNo
+	state := status.Desc.State
+	// Update Pod OpenFlow entries only after the OpenFlow port state is live or down.
+	// Accepting Port state "openflow15.PS_LINK_DOWN" is a workaround for Windows OVS issue https://github.com/openvswitch/ovs-issues/issues/351.
+	// In which OVS does not correctly implement function netdev_windows_update_flags, so OVS doesn't update ifp_flags
+	// after a new OpenFlow port is successfully installed. Since this OVS issue doesn't have side impact on datapath
+	// packets forwarding, antrea-agent will ignore the bad state to ensure the Pod's OpenFlow entries are installed as
+	// long as the port number is allocated.
+	if status.Desc.State != openflow15.PS_LIVE && status.Desc.State != openflow15.PS_LINK_DOWN {
```

Comment:
Not sure why PS_LINK_DOWN is used here What's the possible state list? I saw the 'OFPUTIL_PS_BLOCKED' is mentioned in the issue 351. What will be the state retrieved by Antrea when the OVS port state is “OFPUTIL_PS_BLOCKED”?

Reply:
It should be `PS_LINK_DOWN` after comparing the openflow message captured from live setup